node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,7 @@
+import { readFile } from 'node:fs/promises';
+try {
+  const file = await readFile(`${process.argv[2]}`, 'utf8');
```

Comment:
There's no need to use string interpolation here. You can just pass `process.argv[2]` directly.